.azure-pipelines/docker-sonic-slave.yml

```diff
@@ -12,26 +12,27 @@ resources:
     endpoint: sonic-net
 
 schedules:
-- cron: "0 8 * * *"
+- cron: "0 0 * * 0"
```

Comment:
Can we use batch build, if any change of the slave Dockerfile, we would like to build the new version of the docker image.

Reply:
added